DataDog/chaos-controller
# Injector mounts rework

### What does this PR do?

It reworks the mount points used in the injector.

### Motivation

The injector needs multiple mounts to apply and clean disruptions. Those mount points were all in RW while some of them are not needed, especially the biggest one mounting the whole host root. With this rework, we now have the root mounted as RO and a set of other mounts in RW when it is needed.

### Testing Guidelines

* `make docker-build`
* `make install`
* Apply any of the example disruptions and it should still work as expected.
